### PR TITLE
Market Plugin: Upped limit for fetch

### PR DESCRIPTION
**Pull request by tanndlin**

### CODE REVIEW

File: src/app/plugins/marketplace.plugin.ts [hunk 1]
```diff
@@ -90,4 +72,35 @@ export class MarketPlacePlugin extends Plugin {
...
+        const { content } = msg;
+
+        //If content does not contains either element of array then stop.
+        if (
+          ![this._NEW_LISTING_PREFIX, this._NEW_ALIAS_PREFIX].some((el) => content.includes(el))
+        ) {
+          return;
+        }
+
+        let [, item] = content.split('add');
```
Comment (c650) [inline src/app/plugins/marketplace.plugin.ts:95]: const

File: src/app/plugins/marketplace.plugin.ts [hunk 2]
```diff
@@ -48,39 +48,21 @@ export class MarketPlacePlugin extends Plugin {
...
+    if (!itemsForSale) {
+      return;
+    }
+
+    const embed = new RichEmbed();
+    embed.setTitle('Items For Sale');
+    embed.setColor('#7289da');
+    embed.setDescription(itemsForSale.reverse().join('\n\n'));
+    this._replyToUser(message, embed);
+    message.delete();
```
Comment (c650) [inline src/app/plugins/marketplace.plugin.ts:65]: does it get deleted instantly?

Comment (tanndlin) [reply src/app/plugins/marketplace.plugin.ts:65]: As soon as the bot replies, to the user its almost instant

File: src/app/plugins/marketplace.plugin.ts (hunk 1 above)
Comment (tanndlin) [reply src/app/plugins/marketplace.plugin.ts:95]: Cant use const, there is a reassign on line 98

File: src/app/plugins/marketplace.plugin.ts (hunk 2 above)
Comment (c650) [reply src/app/plugins/marketplace.plugin.ts:65]: oh, i see, it deletes teh `!market list` ok cool

**Review by c650 — APPROVED** [review]

File: src/app/plugins/marketplace.plugin.ts (hunk 2 above)
Comment (tanndlin) [reply src/app/plugins/marketplace.plugin.ts:65]: Yeah, deletes the command, and the actual thing the bot sends goes to dms

**Pull request merged**